src/PatchKit.Api/ApiConnection.cs

```diff
@@ -167,6 +163,39 @@ protected T ParseResponse<T>(IApiResponse response)
             }
         }
 
+        private IHttpResponse MakeResponse(Uri uri, Request request)
+        {
+            if (string.IsNullOrEmpty(request.Method) || request.Method == "GET")
+            {
+                var httpRequest = new HttpGetRequest
+                {
+                    Address = uri,
+                    Timeout = RequestTimeoutCalculator.Timeout
+                };
+
+                Logger.LogTrace($"timeout = {httpRequest.Timeout}ms");
+
+                return HttpClient.Get(httpRequest);
+            }
+            else if (request.Method == "POST")
+            {
+                var httpRequest = new HttpPostRequest
+                {
+                    Address = uri,
+                    Timeout = RequestTimeoutCalculator.Timeout,
+                    FormData = request.Data
```

Comment:
I think that here's small inconsistency - in the pull request [here](https://github.com/patchkit-net/patchkit-network-dotnet/pull/1/) there's not property called `FormData` (I guess it should be `Query` for which I suggested a new name `Body`)